### PR TITLE
Index administrative tags via DOR API

### PR DESCRIPTION
**Pull request by mjgiarlo**
Unblocks sul-dlss/dor-services-app#709
Part of sul-dlss/dor-services-app#679

As part of this work, stop indexing tags via the identity metadata datastream.

## Why was this change made?

To make sure tags added to objects via API are indexed.

## Was the documentation (README, DevOpsDocs, wiki, consul, etc.) updated?

No.

## Does this change affect how this application integrates with other services?

Yes, and it was tested successfully in stage and now there's an integration test: https://github.com/sul-dlss/infrastructure-integration-test/pull/34

### CODE REVIEW

**Comment by jcoyne** [comment]
Currently this work is performed by `IdentityMetadata#to_solr`, via DataIndexer.  Do we need to turn that code off?

**Comment by mjgiarlo** [comment]
@jcoyne :speech_balloon: 
> Currently this work is performed by `IdentityMetadata#to_solr`, via DataIndexer. Do we need to turn that code off?

This PR now does that, thanks to the work you've done. Thanks!

**Comment by jcoyne** [comment]
So this is currently blocked by https://github.com/sul-dlss/dor-services-client/pull/154 and a release of DSC, right?

**Comment by mjgiarlo** [comment]
@jcoyne :speech_balloon: 
> So this is currently blocked by [sul-dlss/dor-services-client#154](https://github.com/sul-dlss/dor-services-client/pull/154) and a release of DSC, right?

Yes, this is blocked by the DSC work being merged and cut in a release. I am testing all the components now to make sure they work as expected, and then I will unhold the DSC PR.

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ FROM ruby:2.7.0-alpine
 
 RUN apk add --update --no-cache  \
   build-base \
+  git \
```
Comment (jcoyne) [inline Dockerfile]: is this necessary?

Comment (mjgiarlo) [reply Dockerfile]: @jcoyne It was when I was pointing DIA at a GitHub branch. I figured I'd leave it in to make it easier to do this in the future.

Comment (jcoyne) [reply Dockerfile]: But this makes the images substantially bigger, which means pulling this image for testing takes longer.

Comment (mjgiarlo) [reply Dockerfile]: @jcoyne looks like it adds ~16MB to a package that is ~155MB. :man_shrugging: I can remove it.

Comment (jcoyne) [reply Dockerfile]: 10% reduction!

Comment (mjgiarlo) [reply Dockerfile]: Removed

**Pull request merged**